modules/installation-machine-requirements.adoc

```diff
@@ -85,7 +85,7 @@ endif::ibm-z[]
 ====
 
 ifndef::ibm-z,ibm-power[]
-The bootstrap and control plane machines must use {op-system-first} as the operating system. However, the compute machines can choose between {op-system-first}, {op-system-base-full} 8.6, {op-system-base} 8.7, or {op-system-base} 8.8.
+The bootstrap and control plane machines must use {op-system-first} as the operating system. However, the compute machines can choose between {op-system-first}, {op-system-base-full} 8.6—8.9.
```

Comment:
ISG nit: don't use em dashes. Suggestion:
```suggestion
The bootstrap and control plane machines must use {op-system-first} as the operating system. However, the compute machines can choose between {op-system-first}, {op-system-base-full} 8.6, {op-system-base} 8.7, {op-system-base} 8.8, {op-system-base} 8.9.
```
or maybe,
```suggestion
The bootstrap and control plane machines must use {op-system-first} as the operating system. However, the compute machines can choose between {op-system-first}, {op-system-base-full} 8.6, 8.7, 8.8 and {op-system-base} 8.9.
```

Reply:
Hi @jldohmann I got my en and em dash confused. For ranges [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-dashes) and SSG both suggest to use en dashes so I'll update and replace `—` with `-`.  Does that work?

Reply:
sounds good to me!